src/preferences/dialog/dlgprefmixer.cpp

```diff
@@ -220,245 +253,179 @@ void DlgPrefMixer::slotNumDecksChanged(double numDecks) {
-        radioButtonAdditive->setChecked(true);
-    }
-
-    m_xFaderReverse = m_pConfig->getValueString(
-                                       ConfigKey(EngineXfader::kXfaderConfigKey, "xFaderReverse"))
-                              .toInt() == 1;
-    checkBoxReverse->setChecked(m_xFaderReverse);
-
-    slotUpdateXFader();
-
-    // EQ ////////////////////////////////////////////////
-    QString highEqCourse = m_pConfig->getValueString(ConfigKey(kConfigGroup, "HiEQFrequency"));
-    QString highEqPrecise = m_pConfig->getValueString(
-            ConfigKey(kConfigGroup, "HiEQFrequencyPrecise"));
-    QString lowEqCourse = m_pConfig->getValueString(ConfigKey(kConfigGroup, "LoEQFrequency"));
-    QString lowEqPrecise = m_pConfig->getValueString(
-            ConfigKey(kConfigGroup, "LoEQFrequencyPrecise"));
-    m_bEqAutoReset = static_cast<bool>(
-            m_pConfig->getValueString(ConfigKey(kConfigGroup, "EqAutoReset"))
-                    .toInt());
-    CheckBoxEqAutoReset->setChecked(m_bEqAutoReset);
-    m_bGainAutoReset = static_cast<bool>(
-            m_pConfig->getValueString(ConfigKey(kConfigGroup, "GainAutoReset"))
-                    .toInt());
-    CheckBoxGainAutoReset->setChecked(m_bGainAutoReset);
-    CheckBoxBypass->setChecked(
-            m_pConfig->getValue(
-                    ConfigKey(kConfigGroup, kEnableEqs), QString("yes")) == "no");
-    CheckBoxEqOnly->setChecked(
-            m_pConfig->getValue(
-                    ConfigKey(kConfigGroup, kEqsOnly), "yes") == "yes");
-    CheckBoxSingleEqEffect->setChecked(
-            m_pConfig->getValue(
-                    ConfigKey(kConfigGroup, kSingleEq), "yes") == "yes");
-
-    double lowEqFreq = 0.0;
-    double highEqFreq = 0.0;
-
-    // Precise takes precedence over course.
-    lowEqFreq = lowEqCourse.isEmpty() ? lowEqFreq : lowEqCourse.toDouble();
-    lowEqFreq = lowEqPrecise.isEmpty() ? lowEqFreq : lowEqPrecise.toDouble();
-    highEqFreq = highEqCourse.isEmpty() ? highEqFreq : highEqCourse.toDouble();
-    highEqFreq = highEqPrecise.isEmpty() ? highEqFreq : highEqPrecise.toDouble();
-
-    if (lowEqFreq == 0.0 || highEqFreq == 0.0 || lowEqFreq == highEqFreq) {
-        setDefaultShelves();
-        lowEqFreq = m_pConfig
-                            ->getValueString(ConfigKey(
-                                    kConfigGroup, "LoEQFrequencyPrecise"))
-                            .toDouble();
-        highEqFreq = m_pConfig
-                             ->getValueString(ConfigKey(
-                                     kConfigGroup, "HiEQFrequencyPrecise"))
-                             .toDouble();
-    }
-
+void DlgPrefMixer::setDefaultShelves() {
     SliderHiEQ->setValue(
-            getSliderPosition(highEqFreq,
+            getSliderPosition(2500,
                     SliderHiEQ->minimum(),
                     SliderHiEQ->maximum()));
     SliderLoEQ->setValue(
-            getSliderPosition(lowEqFreq,
+            getSliderPosition(250,
```

Comment:
why are these magic constants all of a sudden?

Reply:
They have been before.
There's an offset in the diff: before points to `loadSettings()`, after to `setDefaultShelves()`.